### PR TITLE
Updates the caret position after pasting into the editor.

### LINKED ISSUE #301: When pasting, the cursor does not move to the end of the pasted content
Copy-paste something into the editor. You'll see that the cursor stays in front of the text.

### PR DESCRIPTION
**Pull request by diegoreymendez**
Fixes #301

To test:

Just paste something and make sure the caret position is moved to the end of the pasted content.

### CODE REVIEW

File: Aztec/Classes/TextKit/TextView.swift [hunk 1]
```diff
@@ -141,7 +141,9 @@ open class TextView: UITextView {
         }
 
         string.loadLazyAttachments()
+
         storage.replaceCharacters(in: selectedRange, with: string)
+        selectedRange = NSRange(location: selectedRange.location + string.length, length: 0)
```
Comment (SergioEstevao) [inline Aztec/Classes/TextKit/TextView.swift]: We should call to the TextView delegate to say that the selection changed.

File: Aztec/Classes/TextKit/TextView.swift [hunk 2]
```diff
@@ -156,6 +158,7 @@ open class TextView: UITextView {
         string.loadLazyAttachments()
 
         storage.replaceCharacters(in: selectedRange, with: string)
+        selectedRange = NSRange(location: selectedRange.location + string.length, length: 0)
```
Comment (SergioEstevao) [inline Aztec/Classes/TextKit/TextView.swift]: We should call to the TextView delegate to say that the selection changed.

Comment (SergioEstevao) [inline Aztec/Classes/TextKit/TextView.swift]: I went back to double check, and the textViewDidChangeSelection is actually being called when you change the selecteRange, what is not being called is textViewDidChange.

File: Aztec/Classes/TextKit/TextView.swift (hunk 1 above)
Comment (SergioEstevao) [inline Aztec/Classes/TextKit/TextView.swift]: I went back to double check, and the textViewDidChangeSelection is actually being called when you change the selecteRange, what is not being called is textViewDidChange.

**Comment by diegoreymendez** [comment]
@SergioEstevao - Ready for another look.

**Pull request merged**